src/components/time-picker/time-picker.tsx

```diff
@@ -847,7 +861,11 @@ export class TimePicker {
           <span class={CSS.delimiter}>{this.localizedSecondSuffix}</span>
         )}
         {showMeridiem && (
-          <div class={CSS.column} role="group">
+          <div
+            class={CSS.column}
+            role="group"
+            style={{ order: this.getMeridiemOrder() === 0 ? "-1" : "0" }}
```

Comment:
It may be better to use a class here instead of using inline styles.

I would suggest...

```
class={{
[CSS.column]: true,
[CSS.columnFirst:  meridiemOrder === 0,
[CSS.columnSecond: meridiemOrder !== 0
}}
```